x-pack/plugins/observability/public/pages/cases/header_badge.tsx

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I see that we don't render any html code. Why it has to be a component and not a hook?